internal/pkg/describe/backend_app.go

```diff
@@ -0,0 +1,190 @@
+			ProjectName: d.app.Project,
+		}, env.Name)
+		configs = append(configs, &AppConfig{
+			Environment: env.Name,
+			Port:        appParams[stack.LBWebAppContainerPortParamKey],
+			Tasks:       appParams[stack.AppTaskCountParamKey],
+			CPU:         appParams[stack.AppTaskCPUParamKey],
+			Memory:      appParams[stack.AppTaskMemoryParamKey],
+		})
+		backendAppEnvVars, err := d.appDescriber.EnvVars()
+		if err != nil {
+			return nil, fmt.Errorf("retrieve environment variables: %w", err)
+		}
+		envVars = append(envVars, flattenEnvVars(env.Name, backendAppEnvVars)...)
+	}
+	sort.SliceStable(envVars, func(i, j int) bool { return envVars[i].Environment < envVars[j].Environment })
+	sort.SliceStable(envVars, func(i, j int) bool { return envVars[i].Name < envVars[j].Name })
+
+	resources := make(map[string][]*CfnResource)
+	if d.enableResources {
+		for _, env := range environments {
+			stackResources, err := d.appDescriber.AppStackResources()
+			if err == nil {
+				resources[env.Name] = flattenResources(stackResources)
+				continue
+			}
+			if !IsStackNotExistsErr(err) {
+				return nil, fmt.Errorf("retrieve application resources: %w", err)
+			}
+		}
+	}
+
+	return &backendAppDesc{
+		AppName:          d.app.Name,
+		Type:             d.app.Type,
+		Project:          d.app.Project,
+		Configurations:   configs,
+		ServiceDiscovery: services,
+		Variables:        envVars,
+		Resources:        resources,
+	}, nil
+}
+
+// backendAppDesc contains serialized parameters for a backend application.
+type backendAppDesc struct {
+	AppName          string             `json:"appName"`
+	Type             string             `json:"type"`
+	Project          string             `json:"project"`
+	Configurations   configurations     `json:"configurations"`
+	ServiceDiscovery serviceDiscoveries `json:"serviceDiscovery"`
+	Variables        envVars            `json:"variables"`
+	Resources        cfnResources       `json:"resources,omitempty"`
+}
+
+// JSONString returns the stringified BackendApp struct with json format.
+func (w *backendAppDesc) JSONString() (string, error) {
+	b, err := json.Marshal(w)
+	if err != nil {
+		return "", fmt.Errorf("marshal applications: %w", err)
+	}
+	return fmt.Sprintf("%s\n", b), nil
+}
+
+// HumanString returns the stringified BackendApp struct with human readable format.
+func (w *backendAppDesc) HumanString() string {
+	var b bytes.Buffer
+	writer := tabwriter.NewWriter(&b, minCellWidth, tabWidth, cellPaddingWidth, paddingChar, noAdditionalFormatting)
+	fmt.Fprintf(writer, color.Bold.Sprint("About\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\n", "Project", w.Project)
+	fmt.Fprintf(writer, "  %s\t%s\n", "Name", w.AppName)
+	fmt.Fprintf(writer, "  %s\t%s\n", "Type", w.Type)
+	fmt.Fprintf(writer, color.Bold.Sprint("\nConfigurations\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\n", "Environment", "Tasks", "CPU (vCPU)", "Memory (MiB)", "Port")
```

Comment:
this is so much better thank you! Do you think we should move the column names and section titles to the methods too?

Reply:
I have no idea actually but to make it even simpler maybe we should do.

Reply:
Maybe not...I feel like it makes it more easier to read to leave those in humanstring